config/environments/development.rb

```diff
@@ -53,7 +53,7 @@
   # Raises error for missing translations
   config.action_view.raise_on_missing_translations = true
 
-  config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
+  config.action_mailer.default_url_options = { host: config.wcrs_back_office_url, protocol: "http" }
```

Comment:
Do we need these settings to be merged? Are they part of the work here?

Reply:
Yep, see comment on https://github.com/DEFRA/waste-carriers-back-office/pull/574/commits/f5f3831d5f996e7b3c6bf45a34fac6a0a9b7f7d7! Otherwise this didn't work locally.